client/site-profiler/components/site-profiler.tsx

```diff
@@ -48,6 +51,10 @@ export default function SiteProfiler() {
 
 			{ data && (
 				<LayoutBlock className="domain-result-block">
+					{
+						// Translators: %s is the domain name searched
+						<DocumentHead title={ translate( '%s < Site Profiler', { args: [ domain ] } ) } />
```

Comment:
I've tested the existing pages and how we arrange the page title, and for the home page, "My Home" there is a different connection sign, "<". 

Can we adjust it to be the same?

See attachments:

<img width="471" alt="Screenshot 2023-09-26 at 10 09 53" src="https://github.com/Automattic/wp-calypso/assets/1241413/32408a7a-12f4-48ba-ac69-5cf8d113646d">
<img width="454" alt="Screenshot 2023-09-26 at 10 09 40" src="https://github.com/Automattic/wp-calypso/assets/1241413/673f953e-8c25-4d16-a565-6a9d113d00d2">